src/utils/MemoryLockWrapper.h

```diff
@@ -30,6 +30,10 @@ class MemoryLockGuard {
         }
     }
 
+    MemoryLockGuard(MemoryLockGuard&& lg)
+        : lock_(lg.lock_), keys_(std::move(lg.keys_)), locked_(lg.locked_) {}
+
+
```

Comment:
Add the `operator=(MemoryLockGuard&&)` well, `nocept` is necessary as well. Forbid the copy ctor and operator=.

Reply:
> Add the `operator=(MemoryLockGuard&&)` well, `noexcept` is necessary as well. Forbid the copy ctor and operator=.

Assent!